src/components/CallView/CallView.vue

```diff
@@ -13,10 +13,10 @@
 			:local-shared-data="localSharedData" />
 
 		<template v-else>
-			<EmptyCallView v-if="!callParticipantModels.length && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
+			<EmptyCallView v-if="!callParticipantModels.length && !screenSharingActive" :is-sidebar="isSidebar" />
 
 			<div id="videos">
-				<div v-if="!isGrid" class="video__promoted" :class="{'full-page': showFullPage}">
+				<div v-if="!isGrid || !callParticipantModels.length" class="video__promoted" :class="{'full-page': showFullPage}">
```

Comment:
This is because stripe relies on `video__promoted` class and it is needed for the fake speaker view in group conversation  (`isGrid` remains truthy) too. 

Another alternative is to condition style `flex-direction: column` for `#videos` and not to show this video promoted div but there is a high risk of breaking something.

Reply:
I agree that this small change would be sufficient (though it might still jump from speaker to grid view with the first joined participant)

Reply:
yeah, but in both cases, it will have the same risk of view jumping from gid to speaker. Though, it's not visible at the moment.